src/core/server/saved_objects/import/extract_errors.ts

```diff
@@ -37,11 +66,14 @@ export function extractErrors(
         originalSavedObject &&
         originalSavedObject.attributes &&
         originalSavedObject.attributes.title;
+
       if (savedObject.error.statusCode === 409) {
+        // pick the correct title of the saved object that conflicts
+        const realTitle = savedObjectsTitlesMap.get(`${savedObject.type}:${savedObject.id}`);
         errors.push({
           id: savedObject.id,
           type: savedObject.type,
-          title,
+          title: realTitle,
```

Comment:
When the import API returns an error, I think the correct response is to include the title of the object that failed to import. The API is saying "I failed while trying to import the title you gave me". Perhaps we can add a new field to the response like `existingTitle`.